<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="1" hostname="Anands-iMac.local" name="ioslocalparalleltesting.IOSApp" tests="3" failures="0" timestamp="31 Oct 2019 04:17:06 GMT" time="9.415" errors="1">
  <testcase name="testiOSApp" time="0.000" classname="ioslocalparalleltesting.IOSApp">
    <skipped/>
  </testcase> <!-- testiOSApp -->
  <testcase name="testiOSApp" time="9.395" classname="ioslocalparalleltesting.IOSApp"/>
  <testcase name="startTest" time="0.020" classname="ioslocalparalleltesting.IOSApp">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at java.util.TreeMap.rotateLeft(TreeMap.java:2224)
at java.util.TreeMap.fixAfterInsertion(TreeMap.java:2291)
at java.util.TreeMap.put(TreeMap.java:582)
at org.openqa.selenium.AbstractCapabilities.setCapability(AbstractCapabilities.java:98)
at org.openqa.selenium.MutableCapabilities.setCapability(MutableCapabilities.java:100)
at org.openqa.selenium.MutableCapabilities.setCapability(MutableCapabilities.java:83)
at ioslocalparalleltesting.IOSApp.startTest(IOSApp.java:37)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:458)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:168)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:105)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </error>
  </testcase> <!-- startTest -->
</testsuite> <!-- ioslocalparalleltesting.IOSApp -->
